### PR TITLE
Update share link to use next

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description
Update share link to use next
#### PR Checklist

- [ ] CHANGELOG.md updated
- [ ] Documentation added
- [ ] Tests updated
- [ ] Config converters updated

### CODE REVIEW

File: docs/sources/operator/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ weight: 300
 
 # Static mode Kubernetes operator (Beta)
 
-{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<ALLOY_VERSION>">}}
+{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<next>">}}
```
Comment (rfratto) [inline docs/sources/operator/_index.md:16]: Should be `next` and not `<next>`, right?

File: docs/sources/static/_index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ weight: 200
 
 # Static mode
 
-{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<ALLOY_VERSION>">}}
+{{< docs/shared lookup="agent-deprecation.md" source="alloy" version="<next>">}}
```
Comment (rfratto) [inline docs/sources/static/_index.md:13]: Ditto as above

**Review by rfratto — APPROVED** [review]
LGTM % feedback

File: docs/sources/operator/_index.md (hunk 1 above)
Comment (clayton-cornell) [reply docs/sources/operator/_index.md:16]: Oooops... I forgot to push the fix

**Review by jdbaldry — APPROVED** [review]
LGTM, thanks for coming back to this :)

**Pull request merged**